### PR TITLE
Zernike features

### PR DESCRIPTION
**Pull request by angrauma**
Add implementation and test of zernike features.

@dietzc do you mind taking a look?

### CODE REVIEW

**Comment by dietzc** [comment]
@angrauma any comments on this?

**Comment by dietzc** [comment]
@angrauma  can you also change them according to first order statistics, haralick and moments (see master)?

**Comment by angrauma** [comment]
Modified the zernike implementation, using now the center of the given region as origin of the circle. @dietzc and @LeeKamentsky, can you please have a look at it.

File: src/main/java/net/imagej/ops/DefaultOpService.java [hunk 1]
```diff
@@ -201,12 +201,14 @@ public void setPriority(final double priority) {
...
+
 	}
 
 	@Override
 	public void setInfo(final PluginInfo<?> info) {
 		this.info = info;
 	}
+	
 
 	// -- Comparable methods --
```
Comment (dietzc) [inline src/main/java/net/imagej/ops/DefaultOpService.java]: Format commit. Please get rid of it ;-)

**Comment by dietzc** [comment]
as soon as the format commit is gone I think its ready to merge. @LeeKamentsky do you agree? Thanks for reviewing btw!

**Comment by LeeKamentsky** [comment]
Yup, this looks pretty good. I think you will like these features - they're great as inputs for a classifier for certain types of cell images.

**Pull request merged**